task_test.go

```diff
@@ -531,6 +531,14 @@ func (s *TaskSuite) TestAddHosts() {
 	s.NoError(err)
 }
 
+func (s *TaskSuite) TestImagePlatform() {
+	s.req.Config.ContextDir = "testdata/basic"
+	s.req.Config.ImagePlatform = "linux/arm64"
```

Comment:
is there a way to verify the built artifact?

Reply:
There is! Update the test to verify the OS and architecture.